sympy/assumptions/tests/test_satask.py

```diff
@@ -143,15 +143,9 @@ def test_rational_irrational():
 
     assert satask(Q.irrational(x*y*z), Q.irrational(x) & Q.irrational(y) &
         Q.rational(z)) is None
-    assert satask(Q.irrational(x*y*z), Q.irrational(x) & Q.rational(y) &
-        Q.rational(z)) is True
-    assert satask(Q.irrational(pi*x*y), Q.rational(x) & Q.rational(y)) is True
```

Comment:
Why are these being removed?

Reply:
Since the factbase of satask is simplified, some queries cannot be directly computed by `satask()` and
thus tests for these queries are removed. However, these queries are still computed by `ask()`.

Reply:
Maybe I don't understand what it is that you have changed. I thought that the idea would just be for `satask` to convert incoming predicates like `Q.nonnegative(x)` into primitives like `Q.positive(x) | Q.zero(x)`.

Why has satask's handling of rational/irrational changed?

Are these tests also being tested for `ask` somewhere?

Reply:
My bad! This was not because of reduced factbase. It was because facts from sathandlers were not converted to primitive predicates. It's fixed now.